<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>From</class>
 <widget class="QWidget" name="From">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="boardChildListWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>291</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mapLayout"/>
  </widget>
  <widget class="QPushButton" name="endDriveButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>113</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>운행종료</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleName">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>운행중입니다</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleName_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>탑승중인 아이 목록</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
